templates/shared/forms/interactive/containers.html

```diff
@@ -37,8 +37,15 @@ <h2 class="p-heading--1" id="modal-title">Get in touch with the container team</
                 <label for="company" class="is-required">Company:</label>
                 <input required id="company" name="company" maxlength="255" type="text" />
 
-                <label for="company" class="is-required">Job title:</label>
-                <input required id="title" name="title" maxlength="255" type="text" />
+                <label for="jobtitle" class="is-required">Job title:</label>
```

Comment:
These changes weren't requested in the linked copy doc, is this for another copy update?

Reply:
My bad, they slipped from another task.
Thanks for pointing out